src/blocks/custom.rs

```diff
@@ -101,6 +108,42 @@ impl ConfigBlock for Custom {
             custom.signal = Some(convert_to_valid_signal(signal)?);
         };
 
+        if let Some(paths) = block_config.watch_files {
+            // If the signal is not in the valid range we return an error
+            let paths: Vec<PathBuf> = paths.iter().map(|p| PathBuf::from(p)).collect();
+            let tx_inotify = custom.tx_update_request.clone();
+            thread::Builder::new()
+                .name("custom".into())
+                .spawn(move || {
+                    let mut notify = Inotify::init().expect("Failed to start inotify");
+                    for p in paths {
+                        notify
+                            .add_watch(p, WatchMask::MODIFY)
+                            .expect("Failed to watch file");
```

Comment:
I think the initialization code should be moved to the main thread so that it'll be possible to handle this error. `Inotify` is `Send` and `Sync` so this shouldn't be a problem.

Reply:
done, it now returns an error:
```
Configuration error: custom. Cause: Failed to watch file $HOME/test_i3status: No such file or directory (os error 2)
```